lib/preload.js

```diff
@@ -100,14 +100,18 @@ const runContainer = (docker, container) => {
   }
   const removeListener = () => {
     // Set things back to normal.
-    process.stdin.setRawMode(false)
-    process.stdin.pause()
-    process.stdin.removeListener('data', killContainer)
+    if (process.stdin.setRawMode) {
+      process.stdin.setRawMode(false)
+      process.stdin.pause()
+      process.stdin.removeListener('data', killContainer)
+    }
   }
-  process.stdin.resume()
   // Set raw mode to be able to intercept CTRL-C
-  process.stdin.setRawMode(true)
-  process.stdin.on('data', killContainer)
+  if (process.stdin.setRawMode) {
+    process.stdin.resume()
+    process.stdin.setRawMode(true)
+    process.stdin.on('data', killContainer)
```

Comment:
Are you sure `on('data', ...` won't work here? This will apply to anybody running this in Windows, or over SSH connections, and in both cases I imagine users will expect ctrl-c to work as normal.